### PR TITLE
Remove the default for progressive jpeg

### PR DESCRIPTION
**Pull request by jimlambie**
This PR reverts to the previous behaviour of delivering a non-progressive JPEG by default.

### CODE REVIEW

File: dadi/lib/handlers/image.js [hunk 1]
```diff
@@ -97,6 +97,10 @@ const ImageHandler = function (format, req, {
   this.exifData = {}
   this.isExternalUrl = !pathname.indexOf('http://') || !pathname.indexOf('https://')
 
+  this.noCache = this.parsedUrl.original.query &&
+    Object.keys(this.parsedUrl.original.query).includes('cache') &&
```
Comment (eduardoboucas) [inline dadi/lib/handlers/image.js]:
```suggestion
```

Comment (eduardoboucas) [reply dadi/lib/handlers/image.js]: I don't think this line is necessary, is it?

File: dadi/lib/handlers/image.js [hunk 2]
```diff
@@ -97,6 +97,10 @@ const ImageHandler = function (format, req, {
   this.exifData = {}
   this.isExternalUrl = !pathname.indexOf('http://') || !pathname.indexOf('https://')
 
+  this.noCache = this.parsedUrl.original.query &&
```
Comment (eduardoboucas) [inline dadi/lib/handlers/image.js]: Can we name this something a bit more descriptive, like `this.shouldBypassCache`?

File: dadi/lib/handlers/image.js (hunk 1 above)
Comment (jimlambie) [reply dadi/lib/handlers/image.js]: You're right, it was only if the value was a boolean and it isn't. That being said, I committed this by accident!

File: dadi/lib/handlers/image.js (hunk 2 above)
Comment (jimlambie) [reply dadi/lib/handlers/image.js]: And yes.

**Pull request merged**